### DISCUSSION TITLE
User assignment to SP

### DISCUSSION THREAD

**Discussion by Ztnagluu** [JanssenProject/jans#7950] [Q&A] [answered]
Hi,

I have created an SP provider in Gluu and created an user account in Gluu.
How can I now assign a user to the SP?

**Comment by nynymike** [comment]
SAML is typically "SP initiated". Navigate to your SP, hit "Login" and if it's configured properly, it should redirect you to the SAML IDP (the KC instance in your Janssen deployment)! Please test SAML with Jans version 1.1.0-nightly.

**Reply by Ztnagluu** [reply]
Yes, but before trying to login, do we need to assign user to particular SP integrated in GLUU?
Right now, we do not assign any user to SP.
My question is- How can we assign any user to SP and its after that we try to login so that the redirect goes to SP link later.

**Comment by nynymike** [comment]
No... any user should be able to access the SP. Controlling access to the SP is a different issue. You could use the Shibboleth SP, or even an OpenID Connect RP. You could for example use the `requires` syntax of [mod_auth_openidc](https://github.com/OpenIDC/mod_auth_openidc/wiki/Step-up-Authentication). The IDP only issues the identity assertion with user attributes (i.e. "user claims" in OpenID id_token jargon)

**Comment by Ztnagluu** [comment]
HI MIke,

Can you confirm the correct Gluu server which we have in Open source to proceed with testing first.
We have an environment setup but its UI is completely different and we do not have group management option to add groups in the current UI.

**Reply by Ztnagluu** [reply]
Can we get an update on this if any?
The current Gluu UI does not include group management.

**Comment by nynymike** [comment, marked as answer]
Group management is not an IDP feature--it is a WAM feature from 10 years ago. If you want to do old-style WAM you can use the KC component, but you'll have to use the KC admin interface directly.
